### DISCUSSION TITLE
how to get past the error message python.exe' is not executable. Is it the python binary?

### DISCUSSION THREAD

**Discussion by hercegnovi** [bazelbuild/bazel#18179] [General]
Trying to build the latest tensorflow 2.12 on windows 11 with bazel 5.3 and python 3.11, but am stuck on the error below. Python path is correct as are env variables. Any ideas on how to work around this? 

INFO: Repository local_execution_config_python instantiated at: C:/work/bin/tensorflow/WORKSPACE:15:14: in C:/work/bin/tensorflow/tensorflow/workspace2.bzl:962:19: in workspace C:/work/bin/tensorflow/tensorflow/workspace2.bzl:96:27: in _tf_toolchains C:/work/bin/tensorflow/tensorflow/tools/toolchains/remote_config/configs.bzl:6:28: in initialize_rbe_configs C:/work/bin/tensorflow/tensorflow/tools/toolchains/remote_config/rbe_config.bzl:158:27: in _tensorflow_local_config Repository rule local_python_configure defined at: C:/work/bin/tensorflow/third_party/py/python_configure.bzl:279:41: in ERROR: An error occurred during the fetch of repository 'local_execution_config_python': Traceback (most recent call last): File "C:/work/bin/tensorflow/third_party/py/python_configure.bzl", line 212, column 22, in _create_local_python_repository _check_python_bin(repository_ctx, python_bin) File "C:/work/bin/tensorflow/third_party/py/python_configure.bzl", line 145, column 25, in _check_python_bin auto_config_fail("--define %s='%s' is not executable. Is it the python binary?" % ( File "C:/work/bin/tensorflow/third_party/remote_config/common.bzl", line 12, column 9, in auto_config_fail fail("%sConfiguration Error:%s %s\n" % (red, no_color, msg)) Error in fail: Configuration Error: --define PYTHON_BIN_PATH='C:/Python311/python.exe' is not executable. Is it the python binary? ERROR: C:/work/bin/tensorflow/WORKSPACE:15:14: fetching local_python_configure rule //external:local_execution_config_python: Traceback (most recent call last): File "C:/work/bin/tensorflow/third_party/py/python_configure.bzl", line 212, column 22, in _create_local_python_repository _check_python_bin(repository_ctx, python_bin) File "C:/work/bin/tensorflow/third_party/py/python_configure.bzl", line 145, column 25, in _check_python_bin auto_config_fail("--define %s='%s' is not executable. Is it the python binary?" % ( File "C:/work/bin/tensorflow/third_party/remote_config/common.bzl", line 12, column 9, in auto_config_fail fail("%sConfiguration Error:%s %s\n" % (red, no_color, msg)) Error in fail: Configuration Error: --define PYTHON_BIN_PATH='C:/Python311/python.exe' is not executable. Is it the python binary?

**Comment by meteorcloudy** [comment]
This is a question to better ask on https://github.com/tensorflow/tensorflow, but from the error message, you might want to check if `C:/Python311/python.exe` actually exists on your machine.
